keyboards/takashicompany/minidivide/minidivide.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 takashicompany (@takashicompany)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
```

Comment:
Should be moved to info.json.

Reply:
You can use `qmk info -f json -kb takashicompany/minidivide` to assist.

Reply:
thanks. I will try.